client/version-list/Container.tsx

```diff
@@ -32,7 +32,7 @@ export class ActionDispatcher {
   public async asyncGetNodeVersions(): Promise<void> {
     this.dispatch(fetchStarted());
     try {
-      const path = '/api/versions';
+      const path = '/version';
```

Comment:
Why do you change this to `/version`?

`asyncGetNodeVersions()` is a method to retrieve version numbers of Fireworq nodes, whereas `/version` is an API to retrieve the version number of Fireworqonsole.

Reply:
Yes, you're right. Closing this PR.